src/Microsoft.Extensions.Logging.AzureAppServices/Internal/BatchingLogger.cs

```diff
@@ -40,7 +40,22 @@ public void Log<TState>(DateTimeOffset timestamp, LogLevel logLevel, EventId eve
             builder.Append(logLevel.ToString());
             builder.Append("] ");
             builder.Append(_category);
-            builder.Append(": ");
+
+            var scopeProvider = _provider.ScopeProvider;
+            if (scopeProvider != null)
+            {
+                scopeProvider.ForEachScope((scope, stringBuilder) =>
+                {
+                    builder.Append(" => ").Append(scope);
+                }, builder);
+
+                builder.AppendLine(":");
```

Comment:
If we `AppendLine` here should we also add a few spaces like we do in `ConsoleLogger`?

Reply:
It didn't append spaces in case of multiline messages before so I'm reluctant to add it here.

Reply:
Fair enough